core/CronArchive/SegmentArchiving.php

```diff
@@ -174,7 +174,14 @@ public function getReArchiveSegmentStartDate($segmentInfo)
         }
     }
 
-    private function getCreatedTimeOfSegment($storedSegment)
+    /**
+     * Retrieve the created and last edited time as date objects from the supplied segment array
+     *
+     * @param array $storedSegment
+     *
+     * @return array|null
+     */
+    private function getCreatedTimeOfSegment(array $storedSegment): ?array
```

Comment:
Actually this method always returns an array.
```suggestion
     * @return array
     */
    private function getCreatedTimeOfSegment(array $storedSegment): array
```